mne/minimum_norm/resolution_matrix.py

```diff
@@ -116,20 +325,64 @@ def get_point_spread(resmat, src, idx, norm=False):
         Forward Operator.
     src : instance of SourceSpaces
         Source space used to compute resolution matrix.
-    idx : list of int
-        Vertex indices for which PSFs or CTFs to produce.
-    norm : bool
-        Whether to normalise to maximum across all PSFs (default: False).
+    idx : list of int | list of Label
+        Source for indices for which to compute PSFs or CTFs. If mode is None,
+        PSFs/CTFs will be returned for all indices. If mode is not None, the
+        corresponding summary measure will be computed across all PSFs/CTFs
+        available from idx.
+        Can be:
+
+        * list of integers : Compute PSFs/CTFs for all indices to source space
+          vertices specified in idx.
+        * list of Label : Compute PSFs/CTFs for source space vertices in
+          specified labels.
+
+    mode : None | 'mean' | 'max' | 'svd'
+        Compute summary of PSFs/CTFs across all indices specified in 'idx'.
+        Can be:
+
+        * None : Output individual PSFs/CTFs for each specific vertex
+          (Default).
+        * 'mean' : Mean of PSFs/CTFs across vertices.
+        * 'max' : PSFs/CTFs with maximum norm across vertices. Returns the
+          n_comp largest PSFs/CTFs.
+        * 'svd' : SVD components across PSFs/CTFs across vertices. Returns the
+          n_comp first SVD components.
+    n_comp : int
+        Number of PSF/CTF components to return for mode='max' or mode='svd'.
+        Default n_comp=1.
+    norm : None | 'max' | 'norm'
+        Whether and how to normalise the PSFs and CTFs. This will be applied
+        before computing summaries as specified in 'mode'.
+        Can be:
+
+        * None : Use un-normalized PSFs/CTFs (Default).
+        * 'max' : Normalize to maximum absolute value across all PSFs/CTFs.
+        * 'norm' : Normalize to maximum norm across all PSFs/CTFs.
+    return_svd_vars : bool
+        Whether or not to return the explained variances across the specified
```

Comment:
A more compact version I've found is

"If True (default False), return ..."

Reply:
Good.